FAQ.md

```diff
@@ -0,0 +1,232 @@
+This is a collection of frequently asked questions and the answers, or pointers to them for **Tesseract 4**. [Common errors and information for their resolution](https://github.com/tesseract-ocr/tesseract/wiki/4.0x-Common-Errors-and-Resolutions) is given on a [separate wiki page](https://github.com/tesseract-ocr/tesseract/wiki/4.0x-Common-Errors-and-Resolutions).	------------
+
+
+For the older version of the FAQ pertaining to Tesseract 2.0x, 3.0x and 4.00.00alpha, please see [FAQ Old](FAQ-Old).	**These wiki pages are no longer maintained.** 
```

Comment:
```suggestion
For the older version of the FAQ pertaining to Tesseract 2.0x, 3.0x and 4.00.00alpha, please see [FAQ Old](FAQ-Old.md).	**These wiki pages are no longer maintained.** 
```

Reply:
Do we need to mention the unmaintained wiki pages here?

Reply:
Oh, I think that got in unintentionally when I copied the page from FAQ.md in wiki.

Actually it maybe preferable if you can get it directly from the wiki  (https://github.com/tesseract-ocr/tesseract/wiki/FAQ)  rather than this PR in order to get the history.